pkgs/servers/audiobookshelf/default.nix

```diff
@@ -28,7 +28,7 @@ let
   };
 
   wrapper = import ./wrapper.nix {
-    inherit stdenv ffmpeg-full tone pname nodejs;
+    inherit stdenv ffmpeg-full tone pname nodejs getopt;
```

Comment:
`getopt` tends to not be installed on most systems, they tend to use the `util-linux` implementation. I think it's better to use the `util-linux` package here instead.